src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1785,6 +1816,24 @@ public function setIgnoreMandatoryFlag($ignoreMandatoryFlag)
     }
 
     /**
+     * Orders a child before another child
+     * @param NodeInterface $parent
+     * @param NodeInterface $childSource
+     * @param NodeInterface $childDest
+     */
+    private function executeOrderBefore(
+        NodeInterface $parent,
+        NodeInterface $childSource,
+        NodeInterface $childDest
+    ) {
+        // reorder
+        $parent->orderBefore($childSource->getName(), $childDest->getName());
+
+        $event = new ContentOrderBeforeEvent($childDest, $childSource);
+        $this->eventDispatcher->dispatch(ContentEvents::NODE_ORDER_BEFORE, $event);
```

Comment:
after a discussion with @danrot we decided to change the name of the event to `NODE_ORDER` and pass the event the node, the old index and the new index.

Important: don´t forget to adapt `Sulu\Component\Content\Mapper\Subscriber\NodeOrderSubscriber` which uses the `NODE_ORDER_BEFORE` to resort the nodes with `sulu:order` property!